### ISSUE TITLE
"Oops, something went wrong," and Powershell told me to put it here...

### ISSUE THREAD

**Issue by jm2801** [PowerShell/PSReadLine#325]
Beginning dump:

---

Last 200 Keys:
 s t \ l a b 1 1 " Enter
 j a v a Spacebar - p c Backspace Backspace c p Spacebar " . ; " C : \ U s e r s \ J o n a t h a n \ D o c u m e n t s \
 B a r d \ 2 0 1 5 Spacebar F a l l , Spacebar S e m e s t e r Spacebar 3 \ D a t a Spacebar S t r u c t u r e s \ i n c
 l u d e " LeftArrow UpArrow DownArrow UpArrow DownArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow Lef
tArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow Lef
tArrow LeftArrow LeftArrow LeftArrow LeftArrow Home Home RightArrow RightArrow RightArrow RightArrow RightArrow RightArr
ow RightArrow RightArrow RightArrow RightArrow RightArrow RightArrow RightArrow RightArrow LeftArrow Backspace End Space
bar LeftArrow LeftArrow \ a l g s 4 Backspace r Backspace 4 . j a r End S y m b o l G r a p h Spacebar m o v i e s . t x
 t Spacebar = Enter
 UpArrow Enter
 UpArrow Ctrl+c

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer s
ize in that dimension.
Parameter name: left
Actual value was 212.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.WriteBufferLines(CHAR_INFO[] buffer, Int32& top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender()
   at Microsoft.PowerShell.PSConsoleReadLine.CancelLine(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.CopyOrCancelLine(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignor
eIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)

---

Basically I pressed Ctrl+c because I thought something was running (it wasn't) and Powershell got confused and barfed this up.  Really don't know what happened.  Maybe has something to do with me having just changed the horizontal buffer size...

![image](https://cloud.githubusercontent.com/assets/14919653/11971452/ed2e54bc-a90b-11e5-8388-84af2f9128e6.png)

**Comment by lzybkr** [comment]
Resizing the console causes a variety of problems, I'm resolving this as a dupe of #292
